src/Components/CreateImageWizard/validators.ts

```diff
@@ -78,6 +79,41 @@ export const isSshKeyValid = (sshKey: string) => {
   return isPatternValid;
 };
 
+type PasswordValidationResult = {
+  isValid: boolean;
+  strength: HelperTextVariant;
+};
+
+const isPasswordLengthValid = (password: string): boolean => {
+  if (!password) return true;
+  return password.length >= 6 && password.length <= 128;
+};
+
+const calculatePasswordStrength = (
+  password: string
+): PasswordValidationResult['strength'] => {
+  if (!password) return 'indeterminate';
+
+  let rulesCount = 0;
+  if (/[a-z]/.test(password)) rulesCount++; // Lowercase
+  if (/[A-Z]/.test(password)) rulesCount++; // Uppercase
+  if (/\d/.test(password)) rulesCount++; // Digit
+  if (/\W/.test(password)) rulesCount++; // Symbol
+
+  return rulesCount >= 3 ? 'success' : 'error';
+};
+
+export const isPasswordValid = (password: string): PasswordValidationResult => {
```

Comment:
This function is called `isPasswordValid` and so I would expect it to return a boolean, not an object.

Reply:
changed it thank you